README.md

```diff
@@ -0,0 +1,347 @@
+## Rules Overview
```

Comment:
Should have an initial line for introducing the package. That we add additional sniffs and fixes to squizlabs/php_codesniffer